biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/Logging.scala

```diff
@@ -38,10 +38,10 @@ object Logging {
 
   private[biopet] val errors: ListBuffer[Exception] = ListBuffer()
 
-  def addError(error: String, debug: String = null): Unit = {
+  def addError(error: String, debug: String = null, cause: Exception = null): Unit = {
```

Comment:
I don't see that using null would do any harm here. Do you want also the string to be an Option, this would mean changing this everywhere in the code, debug to Some(debug), I'd prefer it to be null as it was before.